libsolidity/formal/BMC.cpp

```diff
@@ -1195,10 +1194,6 @@ void BMC::checkBooleanNotConstant(
 	std::vector<SMTEncoder::CallStackEntry> const& _callStack
 )
 {
-	// Do not check for const-ness if this is a constant.
-	if (dynamic_cast<Literal const*>(&_condition))
-		return;
```

Comment:
`checkBooleanNotConstant()`'s docstring needs to be updated. It still mentions that the function checks this.

Reply:
I update the docstring. Actually, coming back to this function, I refactored it further to only perform the satisfiability checks and moved reporting of the result to the only caller.